### ISSUE TITLE
too-many-format-args false positive

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#322]
Originally reported by: **Chris Rebert (BitBucket: [cvrebert](http://bitbucket.org/cvrebert), GitHub: @cvrebert?)**

---

Environment:

```
#!bash

$ pylint --version
No config file found, using default configuration
pylint 1.3.1, 
astroid 1.2.1, common 0.62.1
Python 2.7.7 (default, Jun 14 2014, 23:12:13) 
[GCC 4.2.1 Compatible Apple LLVM 5.1 (clang-503.0.40)]
```

Example:

```
#!python

# foobar.py
print '{0}{1[FOO]}'.format(123, {'FOO': 456})
```

False positive:

```
#!bash

$ pylint -r n -f parseable foobar.py 
No config file found, using default configuration
************* Module foobar
foobar.py:1: [E1305(too-many-format-args), ] Too many arguments for format string
```

Code itself works fine:

```
#!bash

$ python foobar.py
123456
$
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/322
